### ISSUE TITLE
Please allow OpenThingMap in limited circumstances to match current OSM community usage

### ISSUE THREAD

**Issue by amandasaurus** [tieguy/collabmark.org#32]
In #28 and #31, the suggestion to ban "similar terms", OpenThingMap. I suggest allowing Open[THING]Map in certain circumstances, based on long standing usage in the OSM community.

What is [OpenCycleMap](https://www.opencyclemap.org/)? It's an OSM map, but they display is focussed on cycling infastructure. What is [OpenRailwayMap](http://www.openrailwaymap.org/)? An OSM map, but which displays railway things. What is [OpenFireMap](http://openfiremap.org/)? An OSM map which displays things to do with fire fighting. etc etc They would be banned by the new trademark rules, but I think they should be allowed. I think lots of people in the OSM community, when hearing about "Open[THING]Map" would presume that it was a map of [THING]s from OSM, and I think many in the OSM community would be OK with that usage, and would not consider it an attempt to steal from OSM (plus we're a pile of overtly logical techies who are terrible at coming up with names 😉).  So I suggest allowing scheme.

Rather than allow any Open[THING]Map without restriction, I suggest it only be allowed if: (i) [THING] is a noun-like word which refers to something that is mapped in OSM. (ii) You are making a map of that subset of OSM. It might be a good idea to limit it to community made, open, maps, or that it must not be massively commerical, and must not try to immitate OSM (So no "OpenRoadMap")

What I'm suggesting would prevent [OpenLitterMap](https://openlittermap.com/) (which doesn't use OSM), [OpenLoveMap](http://openlovemap.de/) (because "Love" is not really a noun. and is not mapped in OSM in that way, but OpenEroticMap would be OK), [OpenTouchMap](http://www.opentouchmap.org/) (because "touch" is not a noun that's mapped in OSM, and this isn't a map of "touch" things from OSM)

**Comment by simonpoole** [comment]
Hi Rory, I think we all share the sentiment, that "it should be possible" the downside is

- your suggestion is fairly complex and likely just going to cause confusion, but it would be totally OK for an internal guideline for allowing use
- typically these remixes are used in domain names, if we don't "own" those we can't control what happens with them in the future (example OpenWeatherMap which was an OSM based project which obviously got sold off to a commercial entity), clawing our rights back is a difficult proposal

In summary I would rather handle these with a one by one internal approval process.

**Comment by amandasaurus** [comment]
I was going to say that it was wrong for require a case by case approval from the OSMF since that gives too much power to the OSMF, but then I properly read [the draft](https://github.com/tieguy/OSM-collabmark.org/blob/OSM-draft/CPM/Collaborative-Mark-Policy-draft.md), and you need pre-approval for lots of things like conferences etc. So I'd be happy with Open[THING]Map being placed on the same footing as those other commonly accepted uses.

I could work on a patch to move "open[thing]map" from "always prohibited" to "maybe allowed, ask us first"

**Comment by xlcrr** [comment]
@rory what part of OpenLitterMap doesn't use OSM?
